openwisp_users/api/views.py

```diff
@@ -198,7 +199,7 @@ def update(self, request, *args, **kwargs):
         )
 
 
-class BaseEmailView(ProtectedAPIMixin, GenericAPIView):
+class BaseEmailView(ProtectedAPIMixin, FilterByParent, GenericAPIView):
```

Comment:
Yeah ok, we're inheriting from this `FilterByParent` now but there's some code here in BaseEmailView which is now redundant and has to be removed, which is not being done in this patch, so the goal of the issue is not being reached yet.